### PR TITLE
Copy Stack Overflow

### PR DESCRIPTION
**Pull request by campbecf**
Sourced from: https://meta.stackexchange.com/questions/240839/the-new-new-be-nice-policy-code-of-conduct-updated-with-your-feedback

I feel that this is a much better starting point for an online community.

## Be nice.

Whether you've come to ask questions, or to generously share what you know, remember that we’re all here to learn, together. Be welcoming and patient, especially with those who may not know everything you do. Oh, and bring your sense of humor. Just in case.

That basically covers it. But these three guidelines may help:

1. Rudeness and belittling language are not okay. Your tone should match the way you'd talk in person with someone you respect and whom you want to respect you. If you don't have time to say something politely, just leave it for someone who does.

2. Be welcoming, be patient, and assume good intentions. Don't expect new users to know all the rules — they don't. And be patient while they learn. If you're here for help, make it as easy as possible for others to help you. Everyone here is volunteering, and no one responds well to demands for help.

3. Don't be a jerk. These are just a few examples. If you see them, flag them:

  * Name-calling. Focus on the post, not the person. That includes terms that feel personal even when they're applied to posts (like "lazy", "ignorant", or "whiny").
  * Bigotry of any kind. Language likely to offend or alienate individuals or groups based on race, gender, sexual orientation, religion, etc. will not be tolerated. At all. (Those are just a few examples; when in doubt, just don't.)
  * Inappropriate language or attention. Avoid vulgar terms and anything sexually suggestive. Also, this is not a dating site.
  * Harassment and bullying. If you see a hostile interaction, flag it. If it keeps up, disengage — we'll handle it. If something needs staff attention, you can use the contact us link at the bottom of every page.

We're proud to be a large, user-driven space on the internet where name-calling, harassment, and other online nastiness are almost non-existent. It's up to all of us to keep it that way.

In summary, have fun, and be good to each other.

### CODE REVIEW

**Comment by campbecf** [comment]
Closing this in favor of using the Geek Feminism Code of Conduct based on our discussions on Slack.
